doc/contributing/release-process.rst

```diff
@@ -7,517 +7,502 @@ new CKAN release.
-
-   f. Run msgfmt checks::
+      code will be deleted (along with their translations)::
 
-          find ckan/i18n/ -name "*.po"| xargs -n 1 msgfmt -c
-
-      You must correct any errors or you will not be able to send these to Transifex.
-
-      A common problem is that Transifex adds to the end of a po file as
-      comments any extra strings it has, but msgfmt doesn't understand them. Just
-      delete these lines.
-
-   g. Run our script that checks for mistakes in the ckan.po files::
-
-        ckan -c |ckan.ini| translation check-po ckan/i18n/*/LC_MESSAGES/ckan.po
-
-      If the script finds any mistakes then at some point before release you
-      will need to correct them, but it doesn't need to be done now, since the priority
-      is to announce the call for translations.
-
-      When it is done, you must do the correction on Transifex and then run
-      the tx pull command again, don't edit the files directly. Repeat until the
-      script finds no mistakes.
+        python setup.py update_catalog --no-fuzzy-matching
 
-   h. Edit ``.tx/config``, on line 4 to set the Transifex 'resource' to the new
-      major release name (if different). For instance v2.10.0, v2.10.1 and v2.10.2
+   e. Edit ``.tx/config``, on line 4 to set the Transifex 'resource' to the new
+      major or minor version. For instance v2.10.0, v2.10.1, v2.10.2, etc
       all share: ``[o:okfn:p:ckan:r:2-10]``.
 
-   i. Create a new resource in the CKAN project on Transifex by pushing the new
-      pot and po files::
+   f. Create a new resource in the CKAN project on Transifex by pushing the new
+      pot and po files. Because it reads the new version number in the
+      ``.tx/config`` file, tx will create a new resource on Transifex
+      rather than updating an existing resource ::
 
         tx push --source --translations --force
 
-      Because it reads the new version number in the ``.tx/config`` file, tx will
-      create a new resource on Transifex rather than updating an existing
-      resource (updating an existing resource, especially with the ``--force``
-      option, can result in translations being deleted from Transifex).
-
-      If you get a 'msgfmt' error, go back to the step where msgfmt is run.
+   g. On Transifex give the new resource a more friendly name. Go to the
+      resource (e.g. https://www.transifex.com/okfn/ckan/2-11/) and access the settings
+      from the triple dot icon "...". Keep the slug "2-11", but change
+      the name to "CKAN 2.11".
 
-   j. On Transifex give the new resource a more friendly name. Go to the
-      resource e.g. https://www.transifex.com/okfn/ckan/2-5/ and the settings are
-      accessed from the triple dot icon "...". Keep the slug like "2-4", but change
-      the name to be like "CKAN 2.5".
-
-   k. Update the ``ckan.mo`` files by compiling the po files::
+   h. Update the ``ckan.mo`` files by compiling the po files::
 
         python setup.py compile_catalog
```

Comment:
This looks like it's repeated below, can we remove this one?

Reply:
This section is only needed when starting a new major or minor release so it might be skipped in a patch release, where we also need to pull translations and compile them. I'd rather keep it